charts/fluent-operator/templates/fluentbit-fluentBit.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if .Values.Kubernetes -}}
+apiVersion: fluentbit.fluent.io/v1alpha2
+kind: FluentBit
+metadata:
+  name: fluent-bit
+  labels:
+    app.kubernetes.io/name: fluent-bit
+spec:
+  image: {{ .Values.fluentbit.image.repository }}:{{ .Values.fluentbit.image.tag }}
+  {{- if .Values.fluentbit.imagePullSecrets }}
+  imagePullSecrets:
+{{ toYaml .Values.fluentbit.imagePullSecrets | indent 4 }}
+  {{- end }}
+  positionDB:
+    hostPath:
+      path: /var/lib/fluent-bit/
+  resources:
+{{- toYaml .Values.fluentbit.resources | nindent 4  }}
+  fluentBitConfigName: fluent-bit-config
+  tolerations:
+    - operator: Exists
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+          - matchExpressions:
+              - key: node-role.kubernetes.io/edge
+                operator: DoesNotExist
+  {{- if .Values.fluentbit.secrets }}
+  secrets:
+{{ toYaml .Values.fluentbit.secrets | indent 4 }}
+  {{- end }}
+  {{- if .Values.fluentbit.additionalVolumes }}
+  volumes:
+{{ toYaml .Values.fluentbit.additionalVolumes | indent 4 }}
+  {{- end }}
+  {{- if .Values.fluentbit.additionalVolumesMounts }}
+  volumesMounts:
+{{ toYaml .Values.fluentbit.additionalVolumesMounts | indent 4 }}
+  {{- end }}
+  {{- if .Values.fluentbit.annotations }}
+  annotations:
+{{ toYaml .Values.fluentbit.annotations | indent 4 }}
+  {{- if .Values.fluentbit.output.stdout }}
+    fluentbit.io/exclude: "true"
```

Comment:
@patrick-stephens All changes are done. The stdout output will be enabled by default and other outputs will be disabled by default for fluentbit. Whenever stdout output is enabled, `fluentbit.io/exclude: "true"` annotation will be added to fluentbit pods